std_msgs/Header header
uint8 N_CHANNELS = 16
ChannelInput roll
ChannelInput pitch
ChannelInput yaw
ChannelInput thrust
ChannelState[16] channel_states